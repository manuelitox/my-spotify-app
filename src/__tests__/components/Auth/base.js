import React from 'react'
import { shallow } from 'enzyme'

import AuthPopUp from 'components/Auth/base'

describe('Component | AuthPopUp', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <AuthPopUp />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})