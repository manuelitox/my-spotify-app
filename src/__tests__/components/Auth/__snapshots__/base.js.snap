// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | AuthPopUp should render a component 1`] = `
<OnClickOutside(PopUpContainer)
  WithCloseButton={false}
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  excludeScrollbar={false}
  isOpen={true}
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
  togglePopUp={[Function]}
>
  <Title
    color="#262D45"
    insidePopUp={true}
  >
    Spotify Authorization
  </Title>
  <styled.div>
    <a
      href="https://accounts.spotify.com/authorize?client_id=undefined&redirect_uri=undefined&scope=user-read-private%20user-read-email&response_type=token&state=123"
    >
      Get Authorization
    </a>
    <p>
      <small>
        You need to login into Spotify
      </small>
    </p>
  </styled.div>
</OnClickOutside(PopUpContainer)>
`;
